tests/unittests/test_timeout.py

```diff
@@ -50,16 +56,19 @@ def test_timeout_string_value_passed_in_config(self):
             "client_id": "test_client_id",
             "client_secret": "test_client_secret",
             "refresh_token":"test_refresh_token",
+            "access_token":"test_access_token",
             "user_agent": "test_user_agent",
             "request_timeout": "100"
         }
 
         # initialize 'LinkedinClient'
-        cl = client.LinkedinClient(client_id=config['client_id'],
-        client_secret=config['client_secret'],
-        refresh_token=config['refresh_token'],
-        user_agent=config['user_agent'],
-        request_timeout=config.get('request_timeout'))
+        cl = client.LinkedinClient(
+            client_id=config['client_id'],
+            client_secret=config['client_secret'],
+            refresh_token=config['refresh_token'],
+            access_token=config['access_token'],
+            user_agent=config['user_agent'],
+            request_timeout=config.get('request_timeout'))
```

Comment:
Please align them like this:
```suggestion
        cl = client.LinkedinClient(client_id=config['client_id'],
                                   client_secret=config['client_secret'],
                                   refresh_token=config['refresh_token'],
                                   access_token=config['access_token'],
                                   user_agent=config['user_agent'],
                                   request_timeout=config.get('request_timeout'))
```

Reply:
Made the changes, please re-review.